content/posts/2021/01/2021-06-14-outreachy-application-process/index.md

```diff
@@ -0,0 +1,28 @@
+
+The first step of the application process is the initial/eligibility application. This consists of a number of questions which help Outreachy organizers determine if you are eligible for Outreachy mainly based on on your availability during the internship period and if you are underreprented in tech spaces in your area.  I submitted my initial application had to wait a few days as my proposal was reviewed and decided on if I was eligible for the program. After I got the go ahead, I looked through the projects  to find the ones I could apply to. I selected three projects that I had interest on. I started looking in the code base and entered their communication channels to find more information about the projects. Other projects were still being added to the website and about two weeks in, I saw that Public Lab plots2 project was added in the list of projects.
+
+I had an experience with Public Lab community before, I researched further and totally resonated with what they do. Public Lab is a community and non-profit democratizing science to address environmental issues that affect people. It is a platform where you can post your research, open source it for others to use and also find information you need to solve your environmental and social injustices. There is so much more on [Public lab](http://publiclab.org/). Plus the code community is welcoming and ready to help. I love what Public Lab does and after I saw it listed I knew that is exactly what I would like to work on so I focused on it. You are required to pick 1-2 projects and apply to.
+
+To be eligible for selection to a project, you have to make contributions to the project, there is around three weeks set aside for making contributions. I browsed through the issues and started working on some. The code community was supportive throughout the process. The pull requests got reviewed fast and the community was willing to help in case of any questions/ blockers. I started drafting my proposal early and when the deadline was approching I refined it, recorded my contributions and submitted. 
+
+If you are hesistant to apply to a project, my advice is just go for it. I was quite scared of applying too. The worst thing that could happen is you will contribute to the open source project during the application process and learn a lot in the process if don't get selected. Also all the experience you get from applying could help you improve and reapply the following round.
+
+###  Tips :pencil:
+- Pick a project or two and start contributing to them early enough.
```

Comment:
@cesswairimu I like the fact that you added tips :) , It would also be nice to mention where someone can keep checking if the applications are out or maybe channels to follow to keep updated because in chapter one you mentioned about summer and winter maybe can be much clearer if you also mention the approximate months to expect the applications.

Also,  are these tips for someone who has already passed eligibility and already enrolled into the program?